### DISCUSSION TITLE
Showing all server-speaks-first protocols.

### DISCUSSION THREAD

**Discussion by ahmetgeymen** [linkerd/linkerd2#5214] [Q&A] [answered]
Will be any support for showing the complete list of server-speaks-first protocols. For example I want to know which ports will be allowed or recognized for SMTP. On the website, only port 25 is shown as known SMTP port. May be Linkerd proxies already know port 587 for SMTP too bu I don't know that. I don't want to find out with trying.

Thanks.

**Comment by olix0r** [comment, marked as answer]
Here are all of the ports that are known to be problematic for protocol detection:

https://github.com/linkerd/linkerd2/blob/4e2c86acc464784b37754849d689208564687d7a/charts/linkerd2/values.yaml#L98-L99

There may be others, of course, but 25 and 587 are included in this list already.

**Reply by ahmetgeymen** [reply]
OK, thanks for the answer.
